src/pages/speakers.astro

```diff
@@ -6,11 +6,6 @@ import { peopleBySlug, talksBySlug } from "~/data/notion-data";
 import Speaker from "~/components/Speaker.astro";
 import BodyText from "~/components/BodyText.astro";
 import SafeMarkdown from "~/components/SafeMarkdown.astro";
-
-const speakersToExclude = new Set([
-	// Sylwia might not be able to make it 😢
-	"sylwia-vargas",
-]);
```

Comment:
Sad to see the - on Sylwia, but happy to see the removal of code logic in general!